frontend/app/forms/bank/filtered-forms.component.js

```diff
@@ -24,9 +24,10 @@ export default class FilteredForms extends React.Component {
     }
   }
   recreateFuseSearch() {
-    const reactChildren = React.Children.toArray(this.props.children).filter(
```

Comment:
The changes here were made by eslint.

Reply:
have you tried running `npx eslint --fix`, this may fix the issues for you.

Reply:
if you made no changes to this file, you could check it out and not include it in this commit. 

The build script runs: 
```
script:
  - yarn lint
  - yarn test
  - yarn build
```
I would reccomend running those and seeing if anything is failing locally for you. (The pre-commit also runs something similar)

Reply:
> have you tried running `npx eslint --fix` or `yarn eslint --fix`, this may fix the issues for you.
Yeah I had tried something similar which fixed any issues created by running **yarn lint**

> ./node_modules/.bin/eslint path --fix

I have no issues locally and I could see how running the above command changed the file. It's only failing this travis-ci test.